plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/TokenFetchingMetastoreClientFactory.java

```diff
@@ -121,4 +148,15 @@ private record DelegationToken(long writeTimeNanos, String delegationToken)
             requireNonNull(delegationToken, "delegationToken is null");
         }
     }
+
+    private FailsafeExecutor failSafeExecutor()
+    {
+        RetryPolicy retryPolicy = RetryPolicy.builder()
```

Comment:
If we create this in the constructor, then we don't need to add fields for the retry parameters.